Gems/Atom/RPI/Code/Source/RPI.Reflect/Image/StreamingImageAsset.cpp

```diff
@@ -30,12 +30,13 @@ namespace AZ
                     ;
 
                 serializeContext->Class<StreamingImageAsset, ImageAsset>()
-                    ->Version(1)
+                    ->Version(2)
```

Comment:
Please specify in a few words why the version was changed.

Reply:
Please add a comment following the version number that specifies what the change was. We've found that vastly reduces accidental merge errors.